docs/helm-chart-submission.md

```diff
@@ -45,7 +45,9 @@ For more information on the different Helm Chart Distribution methods, see: [Cre
 
 ## Web catalog only delivery
 
-Web catalog only delivery was previously described as provider delivery. Chart submissions will be made available within the Helm Chart Catalog on successful certifications. In the event that this case is undisirable, the provider should consider the web catalog only option if they wish not to make the chart publicly available. With web catalog only delivery the provider of the chart controls access to the chart and this will impact report generation:
+Web catalog only delivery was previously described as provider delivery. 
+
+Chart submissions will be made available within the Helm Chart Catalog on successful certifications. In the event that this case is undesirable, the provider should consider the web catalog only option if they wish not to make the chart publicly available. With web catalog only delivery the provider of the chart controls access to the chart and this will impact report generation:
```

Comment:
Is the last sentence meant to read like `With web catalog only delivery the provider of the chart controls access to the chart and this will impact report generation:`